vochain/proof.go

```diff
@@ -70,15 +71,26 @@ func VerifyProofOffChainTree(process *models.Process, proof *models.Proof,
 	}
 	switch proof.Payload.(type) {
 	case *models.Proof_Graviton:
-		p := proof.GetGraviton()
-		if p == nil {
-			return false, nil, fmt.Errorf("graviton proof is empty")
-		}
-		valid, err := gravitontree.CheckProof(key, []byte{}, censusRoot, p.Siblings)
-		return valid, big.NewInt(1), err
+		return false, nil, fmt.Errorf("graviton proof no longer supported")
 	case *models.Proof_Iden3:
 		// NOT IMPLEMENTED
 		return false, nil, fmt.Errorf("iden3 proof not implemented")
+	case *models.Proof_Arbo:
+		p := proof.GetArbo()
+		if p == nil {
+			return false, nil, fmt.Errorf("arbo proof is empty")
+		}
+		var hashFunc arbo.HashFunction = arbo.HashFunctionBlake2b
```

Comment:
same as the other line; use `:=`

Reply:
Same reasoning than in https://github.com/vocdoni/vocdoni-node/pull/267#discussion_r714625848